core/state_processor.go

```diff
@@ -78,7 +79,12 @@ func (p *StateProcessor) Process(block *types.Block, parent *types.Header, state
 	)
 
 	// Configure any stateful precompiles that should go into effect during this block.
-	p.config.CheckConfigurePrecompiles(new(big.Int).SetUint64(parent.Time), block, statedb)
+	err := p.config.ConfigurePrecompiles(new(big.Int).SetUint64(parent.Time), block, statedb)
+	if err != nil {
+		werr := fmt.Errorf("could not configure precompiles: %w", err)
+		log.Error("failed to process the state changes", "err", werr)
+		return nil, nil, 0, err
+	}
```

Comment:
Can we remove the wrapping of this error here? I think it would make sense to wrap this error inside of `ConfigurePrecompiles` if anywhere.

Reply:
done